### PR TITLE
chore: add readme file with mock server description

### PR DESCRIPTION
**Pull request by tarashaliuk**
closes: [HRZ-2671](https://spryker.atlassian.net/browse/HRZ-2671)

[HRZ-2671]: https://spryker.atlassian.net/browse/HRZ-2671?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/fulfillment/readme.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+Run `nx run fulfillment:serve` to start mock server.
+
+### Proxy
+
+Mock server can proxy requests.
+
+- Proxy routes are defined in `mock/proxy-routes.json` file.
+
+- Env variable `ORYX_FULFILLMENT_MOCK_PROXY_URL` is defining the URL where proxy routes will be redirected.
```
Comment (gund) [inline apps/fulfillment/readme.md:16]: I would also add instruction on how to connect mock server to fulfillment app (via the .env file and which env variable).

Comment (AndreyYevtukhov) [reply apps/fulfillment/readme.md:16]: agree, this would be helpful

File: apps/fulfillment/readme.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Fulfillment app
+
+## Mock server
+
+1. Run `nx run fulfillment:init-env:mock` to init env variables for mock server.
+   Variables description:
+   - `ORYX_FULFILLMENT_MOCK_PROXY_URL` - the URL where proxy routes will be redirected
+   - `ORYX_FULFILLMENT_MOCK_PORT` - the port on which the mock server will run (mock server runs on `3000` port if this variable is not defined)
+2. Run `nx run fulfillment:serve` to start mock server.
```
Comment (AndreyYevtukhov) [inline apps/fulfillment/readme.md]: Is it about the mock server or FA?)

Comment (gund) [reply apps/fulfillment/readme.md]: Should be `nx run fulfillment:mock` I think 😁

Comment (tarashaliuk) [reply apps/fulfillment/readme.md]: Oh, missed it, thanks for mentioning :)

**Review by gund — APPROVED** [review]

**Pull request merged**